docs/custom_math/components.md

```diff
@@ -0,0 +1,160 @@
+
+# Math components
+
+Here, we will briefly introduce each of the math components you will need to build an optimisation problem.
+A more detailed description of the math YAML syntax is provided in the [math formulation schema][math-formulation-schema].
+
+## Decision variables
+
+Decision variables (also known as `variables`) are why you are here in the first place.
+They are the unknown quantities whose values will decide the value of the objective you are trying to minimise/maximise under the bounds set by the constraints.
+These include the output capacity of technologies, the per-timestep flow of carriers into and out of technologies or along transmission lines, and storage content in each timestep.
+A decision variable in Calliope math looks like this:
+
+```yaml
```

Comment:
See, I'm OK with using the horizontal scrollbar. The danger with YAML wrapping is that it might break the lines weirdly on particularly narrow screens (e.g., splitting on the `_` of parameter names). I'd prefer sticking with allowing horizontal scrolling but doing better about line breaking _in YAML_ for long descriptions / equations.